brave/brave-core
# Remove FindX from search engines

Fixes brave/brave-browser#2047

- Removed FindX from default search engines.
- Bumped up kBraveCurrentDataVersion to 2.
- Updated unit tests to not reference the removed engine.

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:

1. **Test that FindX is removed from search engines on upgrade:**
- Install a previous version of Brave, start the browser and verify that FindX is listed in the search engines list (do not select FindX as the default search engine - see https://github.com/brave/brave-browser/issues/2047#issuecomment-438708217),
- Install a version of Brave with this fix, start the browser and verify that FindX is no longer listed in the search engines list.

2. **Test that FindX is not listed in a fresh installation:**
- Remove your profile from any prior installation of Brave,
- Install a version of Brave with this fix, start the browser and verify that FindX is not listed in the search engines list.

## Reviewer Checklist:
